docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
 sphinx-argparse
 mock
 sphinx-rtd-theme
+jinja2 < 3.1.0
```

Comment:
I'm a bit confused between this change, and [this one here](https://github.com/maxplanck-ie/snakepipes/pull/829#pullrequestreview-1066579437). Of course, they're on different branches. But still, seems like there might be some inconsistency/ ambiguity...

Reply:
downgrading jinja2 version to < 3.1.0 didn't fix the build of read the docs page. What helped, was upgrading sphinx to 4.x.x (from 1.8.x!), which then automatically pulled the right jinja2 version. I.e. I reverted the jinja2 downgrade and upgraded sphinx instead. This should be the latest status on the develop branch. Are you looking at some older commit here?